tests/third_party/cupy/manipulation_tests/test_rearrange.py

```diff
@@ -0,0 +1,76 @@
+import unittest
```

Comment:
can we extend the scope of public CI with new file?

Reply:
done